spec/sinatra_basic_forms_lab_spec.rb

```diff
@@ -36,9 +36,13 @@
       fill_in(:breed, :with => "Mastiff")
       fill_in(:age, :with => "6 months")
       click_button "submit"
-      expect(page).to have_text("Puppy Name:\nButch")
-      expect(page).to have_text("Puppy Breed:\nMastiff")
-      expect(page).to have_text("Puppy Age:\n6 months")
+
+      expect(page.status_code).to eq(200)
+
+      # The \s below will match any whitespace
+      expect(page).to have_text(/Name:\sButch/i)
+      expect(page).to have_text(/Breed:\sMastiff/i)
```

Comment:
@telegraham should `\s` be quantified 0 or more?

Reply:
@sgharms I love that idea!